docs/source/tasks.rst

```diff
@@ -1565,7 +1565,7 @@ Clips
    - Audio device: MRConfon MKII
 
 The Clips battery stands for an adaptation of (`Nishimoto et al., 2011 <https://doi.org/10.1016/j.cub.2011.08.031>`__), in which participants were to visualize naturalistic scenes edited as video clips of ten and a half minutes each. Each run was always dedicated to the data collection of one video clip at a time. As in the original study, runs were grouped in two tasks pertaining to the acquisition of training data and test data, respectively. Scenes from training-clips (ClipsTrn) task were shown only once. Contrariwise, scenes from the test-clips (ClipsVal) task were composed of approximately one-minute-long excerpts extracted from the clips presented during training. Excerpts were concatenated to construct the sequence of every ClipsVal run; each sequence was predetermined by randomly permuting many excerpts that were repeated ten times each across all runs. The same randomized sequences, employed across ClipsVal runs, were used to collect data from all participants. There were twelve and nine runs dedicated to the collection of the ClipsTrn and ClipsVal tasks, respectively. Data from nine runs of each task were interspersedly acquired in three full sessions; the three remaining runs devoted to train-data collection were acquired in half of one last session, before the `Wedge`_ and `Ring`_ tasks. To assure the same topographic reference of the visual field for all participants, a colored fixation point was always presented at the center of the images. Such point was changing three times per second to ensure that it was visible regardless the color of the movie. Ten and twenty extra seconds of acquisition were respectively added at the beginning and end of every run. The total duration of each run was thus ten minutes and fifty seconds.
-
+Note that, for the test-clips task (ClipsVal), the same images were presented three times to each participants. More precisely, in a given session during which such scenes were shown, three runs showing the same images were acquired. Moreover, the order of these images varied between each run.
```

Comment:
As of now, I understand that some scenes from the training clips were selected and concatenated to create the test clips, and each selected scene was shown once in each of the 3 test runs.
If that's the case, I like this paragraph that you added, I would just rephrase as "more precisely, in a given session, three test runs showing the same images were acquired, with the order of images varying between runs."

Reply:
Do we provide somewhere (Github repo) the image order ? This should be part of the documentation.
LGTM otherwise.

Reply:
> As of now, I understand that some scenes from the training clips were selected and concatenated to create the test clips, and each selected scene was shown once in each of the 3 test runs. If that's the case, I like this paragraph that you added, I would just rephrase as "more precisely, in a given session, three test runs showing the same images were acquired, with the order of images varying between runs."

Actually, I think that images for train and test clips are different (although I'm not absolutely certain).
What I'm trying to explain is the following:
- 9 runs were acquired for test clips
- they were acquired in batches of 3 runs, in 3 different sessions
- in a given session, the 3 test runs will show the same images, but in a different order
- the set of images is different between each session

Reply:
> Do we provide somewhere (Github repo) the image order ? This should be part of the documentation. LGTM otherwise.

I just added links to all indices files available from the `public_protocols` repo.